Modules/xASL_wrp_BIDS2Legacy.m

```diff
@@ -143,7 +126,7 @@
 if ~isempty(SubjectDirs)
     for iDir=1:numel(SubjectDirs)
         currentASL = xASL_adm_GetFileList(SubjectDirs{iDir}, '^ASL4D\.nii$', 'FPListRec');
-        ListASL4D = vertcat(ListASL4D,currentASL);
+        ListASL4D = vertcat(ListASL4D, currentASL);
```

Comment:
Isn't `;` easier than `vertcat`?

Reply:
not sure if this works nicely with cells. But there's nothing wrong with vertcat - it does the job and is the correct and transparent function for that. It can probably handle even better some special cases.